### PR TITLE
Changes for v1.0 release, including announcement

### PR DESCRIPTION
**Pull request by astrofrog**
cc @astropy/astropy-core-maintainers

### CODE REVIEW

File: announcements/release-1.0.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+<meta name="description" content="Astropy. A Community Python Library for Astronomy." />
+<meta name="viewport" content="width=device-width, initial-scale=1" />
+<link rel="shortcut icon" href="favicon.ico" />
+
+<link href='http://fonts.googleapis.com/css?family=Open+Sans:400italic,400,700' rel='stylesheet' type='text/css' />
+<link rel="stylesheet" type="text/css" href="../css/style.css" />
+<link rel="stylesheet" type="text/css" href="../css/jquery.sidr.light.css" />
+
+ 
+<title>Astropy | v0.4 Released!</title>
```
Comment (embray) [inline announcements/release-1.0.html]: Hmm--this could be embarrassing...

File: announcements/release-1.0.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+New and improved major functionality in this release includes:
+<ul>
+<li> Support for Altitude/Azimuth and Galactocentric coordinates in astropy.coordinates
+<li> A new astropy.visualization sub-package
+<li> A new astropy.analytic_functions sub-package
+<li> Support for fitting compound models in astropy.modeling
+<li> Significantly faster C-based readers/writers for astropy.io.ascii
+<li> Support for a new enhanced CSV ASCII table format
+<li> A refactored Table object with improved performance when adding/removing columns
+<li> Support for using Time, Quantity, or SkyCoord arrays as Table columns natively
```
Comment (embray) [inline announcements/release-1.0.html]: Maybe linkify some of these?

File: announcements/release-1.0.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+</ul>
+</p>
+<p>
+In addition, hundreds of smaller improvements and fixes have been made. An overview of the changes is provided at:
+</p>
+<p>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://docs.astropy.org/en/stable/whatsnew/1.0.html">http://docs.astropy.org/en/stable/whatsnew/1.0.html</a>
```
Comment (embray) [inline announcements/release-1.0.html]: Or I guess this is fine.

File: index.html [hunk 4]
```diff
@@ -56,6 +56,7 @@
 			<span class="button">Other Docs ▼</span>
 			<ul class="dropdown">
 				<li><a href="http://devdocs.astropy.org/" target="_blank">In Development</a></li>
+				<li><a href="https://astropy.readthedocs.org/en/v0.4.3/index.html" target="_blank">v0.4.4</a></li>
 				<li><a href="https://astropy.readthedocs.org/en/v0.4.3/index.html" target="_blank">v0.4.3</a></li>
 				<li><a href="https://astropy.readthedocs.org/en/v0.4.2/index.html" target="_blank">v0.4.2</a></li>
 				<li><a href="https://astropy.readthedocs.org/en/v0.4.1/index.html" target="_blank">v0.4.1</a></li>
```
Comment (embray) [inline index.html]: I think in the past we've talked about dropping all but the latest bugfix release (per release series) from this list.  Should we go ahead and do that?

Comment (astrofrog) [inline index.html]: Ok will do that, I agree

Comment (astrofrog) [inline index.html]: In fact we did for versions < 0.4

File: announcements/release-1.0.html (hunk 1 above)
Comment (astrofrog) [inline announcements/release-1.0.html]: Fixed :smile:

**Comment by astrofrog** [comment]
I'm going to go ahead and merge this. The announcement will still be hidden because it's not linked to from the main site (it's just for use on e.g. twitter and facebook to have a 'pretty' version to link to).

**Pull request merged**